src/parsing/parsers/postgresql/jison/select/offset_clause.test.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "namePrefix": "should suggest OFFSET",
+    "beforeCursor": "SELECT * FROM testTable LIMIT 100 ",
+    "afterCursor": "",
+    "noErrors": true,
+    "containsKeywords": ["OFFSET"],
+    "expectedResult": {
+      "lowerCase": false
+    }
+  },
+  {
+    "namePrefix": "should not throw errors",
+    "beforeCursor": "SELECT * FROM testTable OFFSET 100;",
+    "afterCursor": "",
+    "noErrors": true,
+    "containsKeywords": ["SELECT"],
+    "expectedResult": {
+      "lowerCase": false
+    }
+  },
+  {
+    "namePrefix": "should not throw errors",
+    "beforeCursor": "SELECT * FROM testTable LIMIT 1 OFFSET 12; ",
```

Comment:
Created an issue to support queries without ordering `LIMIT` and `OFFSET` https://github.com/gravity-ui/sql-autocomplete-parsers/issues/72

Reply:
Yeah, we need more tests in the future!